particles/List/source/Tiles.js

```diff
@@ -51,28 +51,33 @@ defineParticle(({DomParticle, html, log}) => {
     shouldRender({list}) {
       return Boolean(list);
     }
+    getId(entity) {
+      // TODO(sjmiles): ensure we get the entity.id even if the rawData has it's own id,
```

Comment:
@csilvestrini may have thoughts.

Reply:
I pinged runtime and Myk said "yeah, we should have something for this". I'll file a ticket and reference it before merging.

Reply:
My understanding was that we've intentionally made it very difficult to obtain the entity id (ideally impossible), since we don't want to expose it (it's supposed to be more of an internal runtime thing).

Collision with a real user-defined id field is obviously not great... Maybe we need a reserved prefix or something like that (`__id`?)

Reply:
For more discussion on `id` field: #3147 

If it's really to be opaque, then we have to change Multiplexer. Also, it's unfortunate to have a general way to determine Entity equality (deep compare is a major hassle).